### ISSUE TITLE
eval wav is good, but synthesis wav is bad

### ISSUE THREAD

**Issue by sysuzyx** [Rayhane-mamah/Tacotron-2#299]
eval wav likes this. It's pretty good.
[step-100000-eval-wave-from-linear.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2685325/step-100000-eval-wave-from-linear.zip)
while using `python synthesis.py --GTA==False`, wav sounds bad.
[wav-6-linear.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2685344/wav-6-linear.zip)

Is this because evaluation uses teacher forcing, synthesis not use ground truth alignment?

**Comment by statcom** [comment]
> eval wav likes this. It's pretty good.
> [step-100000-eval-wave-from-linear.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2685325/step-100000-eval-wave-from-linear.zip)
> while using `python synthesize.py --mode=eval --GTA=False`, wav sounds bad.
> [wav-6-linear.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2685344/wav-6-linear.zip)
> 
> Is this because evaluation uses teacher forcing, synthesis not use ground truth alignment?

@sysuzyx  Did you resolve this problem? I am having the same issue. eval sound was similar to human voice, but synthesized sound was not even close.

**Comment by statcom** [comment]
@sysuzyx  I am afraid your model might be a case of overfitting as well as under-training. I had [a similar experience](https://github.com/Rayhane-mamah/Tacotron-2/issues/305). 

At this point, I am not sure its exact cause but it might be because of  multi-GPU based on my tests. Are you using multiple GPUs, too? If answer is yes, try to use single GPU with split_on_cpu=False. I suspect this project does not support multi-GPUs yet.

**Comment by sysuzyx** [comment]
I train it on one GPU. I still think it is because evaluation uses the same teacher forcing rate with training. You can see hparams "natural_eval = False" and also` class TacoTrainingHelper(Helper)` in helpers.py.

**Comment by statcom** [comment]
Yes, I agree that because of the setting, eval wave sounds much better than synthesized wave, but your case seemed to be also affected by overfitting (i.e., large difference between train loss and eval loss).

**Comment by jinfagang** [comment]
What model did u trained? Tacotron-2 or Tacotron, if you trained Tacotron-2 by default, you may also train wavenet, it synethisis with 2 models togather. And tacotron is not very good result..

**Comment by iamxiaoyubei** [comment]
+1. I trained with other datasets. It had good results in training but a bad wav sounds in synthesize.

**Comment by primejava** [comment]
yes ，i have the same question

**Comment by lucasjinreal** [comment]
This implementation has this issue exactly.......... when training on my own dataset, synethisis is bad (better to say random noise). Another implementation by Mozallia seems more reasonable
